src/main/java/mate/academy/hibernate/relations/dao/impl/MovieDaoImpl.java

```diff
@@ -12,11 +15,25 @@ public MovieDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Movie add(Movie movie) {
-        return null;
+        Session session = factory.openSession();
+        Transaction transaction = session.beginTransaction();
```

Comment:
Not sure, but I think they must be inside `try` block.

Reply:
I’m also not sure))) I made it so that there was no need to make them null at first and then initialize them separately